custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -181,9 +181,9 @@ def get_episode_case_structure(self, occurrence_structure):
         kwargs = {
             'attrs': {
                 'case_type': EPISODE_CASE_TYPE,
+                'date_opened': self.patient_detail.pregdate1,
```

Comment:
I do believe reading that particular line of the casexml spec too closely resulted in me bricking a bunch of phones.